test/services/MeasureService.test.ts

```diff
@@ -445,6 +473,34 @@ describe('MeasureService', () => {
           );
         });
     });
+
+    it('throws a 400 error when an id is included in both the path and a fhir parameter', async () => {
```

Comment:
```suggestion
    it('throws a 400 error when an id is included in both the path and a FHIR parameter for $data-requirements operation', async () => {
```
I would add something like that at the end!